core/src/identity/ecdsa.rs

```diff
@@ -222,6 +223,12 @@ impl fmt::Debug for PublicKey {
     }
 }
 
+impl hash::Hash for PublicKey {
+    fn hash<H: hash::Hasher>(&self, state: &mut H) {
+        self.to_bytes().hash(state);
+    }
+}
```

Comment:
This will likely trigger this clippy lint: https://rust-lang.github.io/rust-clippy/master/#derive_hash_xor_eq

Please add a test that the property documented in the lint holds, ideally with a prop test :)